app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelTop.tsx

```diff
@@ -51,7 +55,11 @@ const ConnectionPanelTop = ({
 	const onClickDisconnect = async (serverToDelete: JenkinsServer) => {
 		await analyticsClient.sendAnalytics(
 			AnalyticsEventTypes.UiEvent,
-			AnalyticsUiEventsEnum.DisconnectServerName
+			AnalyticsUiEventsEnum.DisconnectServerName,
+			{
+				action: `clicked - ${AnalyticsUiEventsEnum.DisconnectServerName}`,
```

Comment:
Forgot to add these attributes previously which means the analytic events were showing up as something stupid like `disconnectServer disconnectServer` instead of `button clicked - disconnectServer`